modules/ossm-rn-new-features.adoc

```diff
@@ -178,6 +349,29 @@ A new version of the Gateway API custom resource definition (CRD) is now availab
 |For multitenant mesh deployment, all Gateway API CRDs must be present. Use the experimental branch.
 |===
 
+[id="new-features-ossm-2-4-9"]
+== New Features {SMProductName} version 2.4.9
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.12 and later.
+
+=== Component versions for {SMProductName} version 2.4.9
+
+// Release is scheduled for July 10, 2024.
+//Includes 2.6, 2.5.3, 2.4.9 and 2.3.13
+
+|===
+|Component |Version
+
+|Istio
+|1.18.5
+
+|Envoy Proxy
+|1.26.8
+
+|Kiali
+|1.73.8
```

Comment:
also, 1.73.9?

Reply:
There are no component version listed here yet: https://issues.redhat.com/browse/OSSM-6435 so the Component Tables have not been updated.